src/class-parsely.php

```diff
@@ -53,6 +78,7 @@ class Parsely {
 		'metadata_secret'             => '',
 		'parsely_wipe_metadata_cache' => false,
 		'disable_autotrack'           => false,
+		'plugin_version'              => '',
```

Comment:
Added `plugin_version` as non optional field while defining array structure so therefore it also needs the default value.